CIME/Tools/component_generate_baseline

```diff
@@ -44,20 +44,28 @@ OR
         "existing baseline directories to be silently overwritten.",
     )
 
+    parser.add_argument(
+        "--allow-baseline-skip",
+        action="store_true",
+        help="By default an attempt to overwrite an existing baseline directory "
+        "will raise an error. Specifying this option allows tests with "
+        "existing baseline directories to be silently skipped.",
+    )
+
```

Comment:
Add a note that this is incompatible with --allow-baseline-overwrite. And similarly add that this is incompatible with it.

Reply:
Resolveed with 9c8c2b31d.

Reply:
I removed all changes from this file, so resolving.